eng/pipelines/templates/steps/build-artifacts.yml

```diff
@@ -23,8 +23,8 @@ steps:
       inputs:
         pwsh: true
         workingDirectory: $(Build.SourcesDirectory)
```

Comment:
It might be worth pulling this step itself out into a single template we can share as well across the languages. We could of course take in the package and service directory as template parameters.